src/lib.rs

```diff
@@ -86,9 +87,29 @@ impl Position {
     }
 }
 
+impl PartialOrd for Position {
```

Comment:
Just `derive` instead, it will do the right thing since the line field appears before character.

Reply:
@Marwes Thanks for the quick feedback. I didn't know that was ensured. I updated the PR.